tests/common/killring.lisp

```diff
@@ -72,6 +73,26 @@
     (ok (equal '("bazfoobar" (:test3 :test :test2))
                (multiple-value-list (peek-killring-item killring 0))))))
 
+(deftest external-option
+  (let ((killring (make-killring 2)))
+    (with-killring-context (:appending t :before-inserting t)
+      (push-killring-item killring "baz" :options :test))
+
+    ;; clipboard disabled
+    (let ((lem::*enable-clipboard-p* nil)
+          (lem::*killring* killring))
+      (ok (equal '("baz" (:test))
+                 (multiple-value-list (yank-from-clipboard-or-killring)))))
+    
+    ;; clipboard enabled
+    (let ((lem::*enable-clipboard-p* t)
+          (lem::*killring* killring)
+          (expected-result "In LEM we trust."))
+      (lem::invoke-frontend (lambda ()))
```

Comment:
Since you are using lem:invoke-frontend, it may be better to mock it with a fake interface

For example, there is this kind of macro.
https://github.com/lem-project/lem/blob/0c8af91970191d8a997fc82a81eb6d7c4624cd0b/tests/cursors.lisp#L45

Reply:
Thanks, for that information. I was able to update my PR to use this mocking interface 👍.